biggus/__init__.py

```diff
@@ -2560,6 +2726,30 @@ def sub(a, b):
     return _Elementwise(a, b, np.subtract, np.ma.subtract)
 
 
+def multiply(a, b):
+    """
+    Return the elementwise evaluation of `a * b` as another Array.
+
+    """
+    return _Elementwise(a, b, np.multiply, np.ma.multiply)
+
+
+def divide(a, b):
+    """
+    Return the elementwise evaluation of `a / b` as another Array.
+
+    """
+    return _Elementwise(a, b, np.divide, np.ma.divide)
```

Comment:
@pelson & I just discussed this offline. For my own record/sanity, the conclusions we came to are:
 - `divide`, `true_divide`, and `floor_divide` are all simple "wrappers" for the corresponding numpy functions so are consistent with numpy behaviour "for free".
 - `__div__` can always use biggus.floor_divide (NB. It will *only* get called if future-division is *off*.)
 - `__truediv__` can always use biggus.true_divide
 - `__floordiv__` can always use biggus.floor_divide